specs/wp-signup-spec.js

```diff
@@ -183,14 +188,16 @@ test.describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function()
 	);
 
 	test.describe(
-		'Sign up for a site on a premium paid plan through main flow @parallel @visdiff',
+		'Sign up for a site on a premium paid plan through main flow in USD currency @parallel @visdiff',
 		function() {
 			this.bailSuite( true );
 
 			const blogName = dataHelper.getNewBlogName();
 			const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
 			const emailAddress = dataHelper.getEmailAddress( blogName, signupInboxId );
 			const password = config.get( 'passwordForNewTestSignUps' );
```

Comment:
I see this line appears 5 times in different specs. Maybe it worth to extract it to global scope? (defined constants right after imports)

Reply:
done 🐌